code/modules/food_and_drinks/pizzabox.dm

```diff
@@ -323,35 +334,46 @@
 		/obj/item/food/pizza/dank = 0.1,
 	) //pizzas here are weighted by chance to be someone's favorite
 	var/static/list/pizza_preferences
+	boxtag = "Your Favourite"
+	var/mob/living/carbon/human/attuned_to
 
 /obj/item/pizzabox/infinite/Initialize()
 	. = ..()
 	if(!pizza_preferences)
 		pizza_preferences = list()
 
 /obj/item/pizzabox/infinite/examine(mob/user)
+	if(!open && ishuman(user))
+		attune_pizza(user) //this is so our boxtag is always correct
```

Comment:
Can you explain what you are doing here?

Reply:
I explained it with a better comment